schema-reporting-protocol.md

```diff
@@ -46,45 +53,48 @@ type ServiceMutation {
-    * **`graphVariant`**: the graph variant, defaults to `"current"`.
-* _Optional, recommended:_
-    These values will eventually be available on Graph Manager and will allow users to have a better understanding of what was reported, when, and from where.
-    * **`serverId`**: _If available_, an identifier for the instance, such that when restarting this instance it will have the same **`serverId`**, with a different **`bootId`**. In a kubernetes cluster, this might be the **pod** **name**, where the **container** can restart.
-    * `**userVersion**`: an arbitrary string the user can set, so that they could distinguish data on the server by this string. For example this can be the git sha of the repository of the user. We plan to make this visible on Graph Manager.
-* _Optional, for Apollo’s use:_
-    These values will significantly help Apollo improve its service, for example, by enabling us to identify if certain environments/platforms/versions suffer from certain issues.
-    * **`runtimeVersion`**: The runtime in which the edge server is running, e.g. `node 12.03`
-    * **`libraryVersion`**: The library of the running server & reporting agent, e.g. `apollo-server-2.8` , `graphql-java-3.1`
-    * **`platform`**: The infra environment in which this server is running, e.g. `localhost`, `kubernetes/deployment`, `aws lambda`, `google cloud run`, `google cloud function`, `AWS ECS`, etc.
-
-## Edge server side normalization
-
-On the edge server, we’ll expect reporting client to take these steps to ensure a stable schema document:
-
-1. stable sort ordering of type definitions
-2. stable sort ordering of fields in types
-3. stable sort ordering of arguments
-4. remove redundant whitespace
-5. remove comments (not *descriptions*)
-
-Any runtime dependencies of the schema document may result in poor user experience in tracking your schema changes and could result in throttling of service availability.
-
-### Pseudo Code example
+## `EdgeServerInfo` fields
+
+### Required fields
+
+| Name  | Type | Description  |
+|---|---|---|
+| `bootId` | `String!` | A randomly generated UUID that's unique for each instance of your edge server. Set this value on server startup (a given value should not persist across restarts). |
+| `executableSchemaId` | `String!` | A unique identifier for the edge server's schema. Should be the hex representation of the schema document's SHA-256 hash. |
+
+### Recommended fields
+
+| Name  | Type | Description  |
+|---|---|---|
+| `graphVariant` | `String!` | The name of the graph variant to register the schema to. The default value is `current`. |
+| `serverId` | `String` | A randomly generated UUID that's unique for each instance of your edge server. Unlike `bootId`, this value _should_ persist across an instance's restarts. In a Kubernetes cluster, this might be the **pod name**, whereas the **container** can restart. |
```

Comment:
```suggestion
| `serverId` | `String` | A randomly generated ID that's unique for each instance of your edge server. Unlike `bootId`, this value _should_ persist across an instance's restarts. In a Kubernetes cluster, this might be the **pod name**, whereas the **container** can restart. |
```